config/default.json

```diff
@@ -89,5 +89,9 @@
         "port": 5052
       }
     }
+  },
+  "caching": {
+    "enabled": true,
+    "refreshMillis": 5000
```

Comment:
Maybe we should set this to 1000 by default to ensure people generally get what they expect when using the UI?